arch/risc-v/include/common/irq.h

```diff
@@ -134,6 +140,14 @@
 #  error not supported !!!
 #endif
 
+/* REG size */
+
+#ifdef CONFIG_ARCH_RV32
+#define REG_SIZE            1
+#else
+#define REG_SIZE            2
+#endif
```

Comment:
Where this is used? Probably the idea was to define this before `#ifdef CONFIG_ARCH_FPU` and have a next code:
```
#if defined(CONFIG_ARCH_DPFPU)
#  define FPU_REG_SIZE      (2 / REG_SIZE)
#elif defined(CONFIG_ARCH_QPFPU)
#  define FPU_REG_SIZE      (4 / REG_SIZE)
#else
#  if REG_SIZE > 1
#    error not supported !!!
#  else
#    define FPU_REG_SIZE      1
#  endif
#endif
```
??

Reply:
Removed, REG_SIZE is unused